packages/language/src/document.js

```diff
@@ -0,0 +1,741 @@
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.entities.goods - Consumer good entities
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.entities.organizations - Organization
+ *     entities detected from the text. This is only present if detections of
+ *     this type were found.
+ * @param {string[]} callback.annotation.entities.other - Other entities
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.entities.people - People entities
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.entities.places - Place entities
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.entities.unknown - Unknown entities
+ *     detected from the text. This is only present if detections of this type
+ *     were found.
+ * @param {string[]} callback.annotation.sentences - Sentences detected from the
+ *     text.
+ * @param {object[]} callback.annotation.tokens - Parts of speech that were
+ *     detected from the text.
+ * @param {object[]} callback.annotation.tokens.text - The piece of text
+ *     analyzed.
+ * @param {object[]} callback.annotation.tokens.partOfSpeech - A description of
+ *     the part of speech (`Noun (common and propert)`,
+ *     `Verb (all tenses and modes)`, etc.).
+ * @param {object[]} callback.annotation.tokens.partOfSpeechTag - A short code
+ *     for the type of speech (`NOUN`, `VERB`, etc.).
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * document.annotate(function(err, annotation) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // annotation = {
+ *   //   language: 'en',
+ *   //   sentiment: 100,
+ *   //   entities: {
+ *   //     organizations: [
+ *   //       'Google'
+ *   //     ],
+ *   //     places: [
+ *   //       'American'
+ *   //     ]
+ *   //   },
+ *   //   sentences: [
+ *   //     'Google is an American multinational technology company ' +
+ *   //     'specializing in Internet-related services and products.'
+ *   //   ],
+ *   //   tokens: [
+ *   //     {
+ *   //       text: 'Google',
+ *   //       partOfSpeech: 'Noun (common and proper)',
+ *   //       partOfSpeechTag: 'NOUN'
+ *   //     },
+ *   //     {
+ *   //       text: 'is',
+ *   //       partOfSpeech: 'Verb (all tenses and modes)',
+ *   //       partOfSpeechTag: 'VERB'
+ *   //     },
+ *   //     ...
+ *   //   ]
+ *   // }
+ * });
+ *
+ * //-
+ * // To request only certain annotation types, provide an options object.
+ * //-
+ * var options = {
+ *   entities: true,
+ *   sentiment: true
+ * };
```

Comment:
I'm not really sure how I feel about this, it seems a little confusing TBH. I'm assuming it is setup this way because `syntax` is the only one that doesn't have a standalone method?